sync-orders-to-bigQuery/README.md

```diff
@@ -0,0 +1,37 @@
+# Moltin -> BigQuery
```

Comment:
Can we follow the structure of https://github.com/moltin/integration-examples/tree/master/shipping-confirmation-sms?

Also we don't use emojis in the other examples in this repo.

Reply:
Tried to follow that format